docs/gettingstarted/modfiles.md

```diff
@@ -81,67 +46,63 @@ modId = "examplemod1"
+| `modId`         | string  |        **mandatory**        |                                   The unique identifier representing this mod. The id must match `^[a-z][a-z0-9_]{1,63}$` (a string 2-64 characters; starts with a lowercase letter; made up of lowercase letters, numbers, or underscores).                                   | `modId="examplemod"`                                            |
+| `namespace`     | string  |      value of `modId`       |                   An override namespace for the mod. The namespace much match `^[a-z][a-z0-9_.-]{1,63}$` (a string 2-64 characters; starts with a lowercase letter; made up of lowercase letters, numbers, underscores, dots, or dashes). Currently unused.                    | `namespace="example"`                                           |
+| `version`       | string  |            `"1"`            | The version of the mod, preferably in a [variation of Maven versioning][versioning]. When set to `${file.jarVersion}`, it will be replaced with the value of the `Implementation-Version` property in the JAR's manifest (displays as `0.0NONE` in a development environment). | `version="1.20.2-1.0.0"`                                        |
+| `displayName`   | string  |      value of `modId`       |                                                                                    The display name of the mod. Used when representing the mod on a screen (e.g., mod list, mod mismatch).                                                                                     | `displayName"Example Mod"`                                      |
+| `description`   | string  | `'''MISSING DESCRIPTION'''` |                                                                           The description of the mod shown in the mod list screen. It is recommended to use a [multiline literal string][multiline].                                                                           | `description='''This is an example.'''`                         |
+| `logoFile`      | string  |          *nothing*          |                                  The name and extension of an image file used on the mods list screen. The logo must be in the root of the JAR or directly in the root of the source set (e.g. `src/main/resources` for the main source set).                                  | `logoFile="example_logo.png"`                                   |
+| `logoBlur`      | boolean |           `true`            |                                                                                             Whether to use `GL_LINEAR*` (true) or `GL_NEAREST*` (false) to render the `logoFile`.                                                                                              | `logoBlur=false`                                                |
```

Comment:
This explanation isn't really helpful to a beginner or anyone else not sufficiently familiar with OpenGL. It should ideally be explained at a more generic level, i.e. whether interpolation should be used while drawing which blurs the image if the render resolution is higher than the image resolution